students/GKim/lesson02/grid_printer.py

```diff
@@ -0,0 +1,45 @@
+"""
+FYI not my code , got help from you tube... but I understand and like this method.
+"""
+
+def print_row(c): 
+    """
+    function to print + and - using a 
+    for loop .  Using input from user the range is -1 because
+    from the first intial print the  section of the column is already created
+    this all prints accross
+    """
+    print("+", end = " ")  
+    for _ in range(c - 1):
+        print("- - - - +", end = " ")
+    print("- - - - +")
+
+def print_column(c):
+    """
+    prints the "columns" " | " in the grid. just like the row first section is 
+    already created by the input given from user.  The colums print across 
+    """
+    print("|", end = " ")
+    for _ in range(c - 1):
+        print("        |", end = " ")
+    print("        |")
+
+def my_grid(c,r):
```

Comment:
This is close but it doesn't achieve what we are aiming for. This prints out the rows/columns but we are looking for a grid where the rows/columns are the same. We are looking to grind a grid (with same rows/columns) but the **size** of each individual cell is another factor.

Reply:
Ok I will make changes